util/util.go

```diff
@@ -149,3 +150,26 @@ func SetSubtract(a []string, b []string) []string {
 	}
 	return diff
 }
+
+// HasEval determine whether matcher contains function eval
+func HasEval(s string) (bool, error) {
+	return regexp.MatchString(`eval\((.*)\)`, s)
+}
+
+// ReplaceEval replace function eval with the value of its parameters
+func ReplaceEval(s string, rule string) string {
+	reg, _ := regexp.Compile(`eval\((.*)\)`)
```

Comment:
If we are sure it compiles, maybe just use `MustCompile`

Reply:
I've changed it